x-pack/plugins/triggers_actions_ui/public/application/components/prompts/empty_prompt.tsx

```diff
@@ -33,17 +31,6 @@ export const EmptyPrompt = ({
             defaultMessage="Create rule"
           />
         </EuiButton>,
-        <EuiButtonEmpty
```

Comment:
Is it about showing the `documentation` link when there is no rule?

Reply:
Correct, it is within the "no rules" prompt. I believe O11Y does not use this link